packages/xonotic/xonotic.pacscript

```diff
@@ -0,0 +1,33 @@
+name="xonotic"
+pkgname="xonotic"
+pkgver="0.8.6"
+gives="xonotic"
+pkgdesc="The Free and Fast Arena Shooter"
+url="https://dl.unvanquished.net/share/xonotic/release/xonotic-${pkgver}.zip"
+hash="50850f8d800e7499722f6ea61e478e96464a375494b5a24da93aa0598cbe964d"
+breaks=("xonotic-git")
+arch=('amd64')
+maintainer="Herisson <spaceguybox@outlook.com>"
+repology=("project: xonotic")
+
+package() {
+  sudo install -Dm755 "xonotic-linux64-sdl" "${pkgdir}/usr/bin/${pkgname}-sdl"
+  sudo install -Dm755 "xonotic-linux64-glx" "${pkgdir}/usr/bin/${pkgname}-glx"
+  sudo mkdir -p "${pkgdir}/usr/share/applications/"
+  sudo mkdir -p "${homedir}/.xonotic"
+  sudo mkdir -p "${pkgdir}/.xonotic"
+  cp ~/.* ${pkgdir}/.xonotic/
```

Comment:
What is this line doing?

Reply:
that one could probably be deleted

Reply:
you mean like 16 right?

Reply:
No, I meant why are you copying all the hidden folders in the home directory anywhere

Reply:
oh god i didn't see that :sob: it's meant to be ./* as in the xonotic data sorry